lib/runtime/wasmer/test_helpers.go

```diff
@@ -31,11 +33,16 @@ func NewTestInstanceWithTrie(t *testing.T, targetRuntime string, tt *trie.Trie)
 	t.Helper()
 
 	cfg := setupConfig(t, tt, DefaultTestLogLvl, common.NoNetworkRole, targetRuntime)
-	runtimeFilepath, err := runtime.GetRuntime(context.Background(), targetRuntime)
-	require.NoError(t, err)
 
-	r, err := NewInstanceFromFile(runtimeFilepath, cfg)
+	_, err := os.Stat(targetRuntime)
+	if errors.Is(err, os.ErrNotExist) {
+		targetRuntime, err = runtime.GetRuntime(context.Background(), targetRuntime)
```

Comment:
1. Should we move this check within `runtime.GetRuntime`? 🤔 
2. We should document (aka comment the function) that `targetRuntime` can be a constant string defined or a file path, in this case in the `NewTestInstanceWithTrie` comment